build.gradle

```diff
@@ -138,7 +138,7 @@ dependencies {
     // transitive dependencies.
     constraints {
         implementation('org.scala-lang:scala-library:2.13.10')
-        implementation('org.json:json:20230227')
+        implementation('org.json:json:20240205')
```

Comment:
We should try upgrading Sam and see if we can remove this constraint, like in https://github.com/DataBiosphere/terra-billing-profile-manager/pull/404